skops/card/_model_card.py

```diff
@@ -205,6 +215,26 @@ def add_plot(self, **kwargs: str) -> "Card":
             self._figure_paths[plot_name] = plot_path
         return self
 
+    def evaluate(self, metric_values):
+        """Evaluates the model and returns the score and the metric.
+
+        Parameters
+        ----------
+        metric_values: dict
+        Dictionary of metric values.
```

Comment:
```suggestion
            Dictionary of metric values.
```

Reply:
I did it like that and it was bugging me with unexpected indent :(